knowledge-base/create-custom-image-bullets.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Create Custom Image Bullets
+description: Create Custom Image Bullets
+type: how-to
+page_title: Create Custom Image Bullets
+slug: create-custom-image-bullets
+position: 0
+tags: 
+res_type: kb
+---
+
+<table>
+<thead>
+	<tr>
+		<th>Product Version</th>
+		<th>Product</th>
+		<th>Author</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>2020.1.316</td>
+		<td>RadPdfProcessing</td>
+		<td><a href="https://www.telerik.com/blogs/author/martin-velikov">Martin Velikov</a></td>
+	</tr>
+</tbody>
+</table>
```

Comment:
The other KBs have a different structure here.

Reply:
Both structures are valid and produce the same result.